### PR TITLE
✨Update to latest "stable" releases

### PR DESCRIPTION
**Pull request by dims**
`0.1.3` from https://github.com/kubernetes-sigs/cluster-api/releases/ 
`0.2.5` from https://github.com/kubernetes-sigs/cluster-api-bootstrap-provider-kubeadm/releases

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: examples/generate.sh [hunk 1]
```diff
@@ -103,20 +103,20 @@ kustomize build "${SOURCE_DIR}/machinedeployment" | envsubst >> "${MACHINEDEPLOY
 echo "Generated ${MACHINEDEPLOYMENT_GENERATED_FILE}"
 
 # Generate Cluster API provider components file.
-CAPI_BRANCH=${CAPI_BRANCH:-"v0.2.3"}
+CAPI_BRANCH=${CAPI_BRANCH:-"v0.2.5"}
```
Comment (vincepri) [inline examples/generate.sh]: Shouldn't this default to "stable"?

Comment (dims) [reply examples/generate.sh]: ack. will revise some of the version stuff in a later PR

**Pull request merged**